source/images/diagrams/ovirt-architecture.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+"""
+oVirt architecure, using https://diagrams.mingrammer.com/
+"""
+
+
+from diagrams import Cluster, Diagram, Edge
+from diagrams.custom import Custom
+from diagrams.elastic.elasticsearch import Elasticsearch, Kibana
+from diagrams.generic.compute import Rack
+from diagrams.generic.network import Switch
+from diagrams.generic.os import Centos, Windows
+from diagrams.ibm.user import Browser
+from diagrams.onprem.database import PostgreSQL
+from diagrams.onprem.logging import Rsyslog
+from diagrams.onprem.monitoring import Grafana
+from diagrams.onprem.network import Apache, Wildfly
+from diagrams.onprem.storage import Glusterfs
+from diagrams.programming.language import Java, Nodejs, Python
+
+LOGO = "../logos/{product}.png"
+
+with Diagram(
+    "oVirt Architecture",
+    direction="TB",
+    graph_attr={"nodesep": "1", "ranksep": "1", "splines": "compound"}
+):
+
+    with Cluster("Storage Domains"):
+        gluster = Glusterfs("Gluster storage")
+        iscsi = Rack("ISCS Storage")
```

Comment:
iSCSI

also, would it be possible to add ceph? maybe one more box with cinderlib on host going to a ceph "storage box"

Reply:
Done both, let me know if seems good enough.